app/components/LeftPaneEntries/index.tsx

```diff
@@ -724,6 +909,48 @@ function LeftPane(props: Props) {
                         </>
                     </TabPanel>
                 )}
+                <TabPanel
+                    name="visuals"
+                    activeClassName={styles.visualsTab}
+                    retainMount="lazy"
+                >
+                    <Switch
+                        name="hide attachments"
```

Comment:
Let's pass `undefined` for the name if typing permits.

Reply:
Switch component only accepts 'string' as it's name value.